src/components/Holdings.tsx

```diff
@@ -38,15 +38,17 @@ const ReserveAssetGrid = () => {
     isLoadingCelo,
   } = useHoldings();
 
+  console.log({ celo }); 
+
   return (
     <section className="grid grid-cols-2 gap-2 md:gap-x-4 md:gap-y-8 lg:grid-cols-4">
       <Amount
         iconSrc={"/assets/tokens/CELO.svg"}
-        context="Funds in on-chain Reserve contract and in custodyy"
+        context="Funds in on-chain Reserve contract and in custody"
         loading={isLoadingCelo}
         label={"CELO"}
-        units={celo.unfrozen.units + celo.custody.units}
-        value={celo.unfrozen.value + celo.custody.value}
+        units={celo?.unfrozen?.units + celo?.custody?.units}
+        value={celo?.unfrozen?.value + celo?.custody?.value}
```

Comment:
Don't have enough context on what's going on here, but in general the `?` operator should be used as a last resort. And this might require some further refactoring probably. Why do we need to add this here in this PR?

Reply:
This was one of the sources of the client side crashing as a result of the missing data. Even when the new price source was added it sometimes caused the client side to crash still. I agree it might need refactoring, I added them in the interest of getting the reserve site back up quickly.

Reply:
The fetching logic wasn't changed in the refactor, so overhauling the actual business logic would be a separate task. 

Patching for now makes sense in my opinion, though noting a `TODO` with a ticket number for example could be good practice for such a fix